### ISSUE TITLE
implement method, false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1661]
fyi, @sbcgua reproduced as

```abap
INTERFACE lif_writer.
  METHODS stringify.
ENDINTERFACE.

INTERFACE lif_json.
  INTERFACES lif_writer.
  ALIASES stringify FOR lif_writer~stringify.
ENDINTERFACE.

CLASS lcl_json DEFINITION.
  PUBLIC SECTION.
    INTERFACES lif_json.
ENDCLASS.

CLASS lcl_json IMPLEMENTATION.
  METHOD lif_json~stringify.
  ENDMETHOD.
ENDCLASS.
```

![image](https://user-images.githubusercontent.com/5888506/104122385-0106be00-5345-11eb-8403-17c0862e6907.png)
